packages/dashboard/src/scss/main.scss

```diff
@@ -262,6 +262,10 @@
     width: 100%;
   }
 
+  .cove-dashboard-filters-container {
+    z-index: 5;
```

Comment:
why is this necessary?

Reply:
Without this, the graph has a higher z-index and will cover the dropdown when it is opened. Since this isn't an html Select element, the z-index is set to be like a div and the proceeding div covers it up.